### DISCUSSION TITLE
Carregar tabelas filhas?

### DISCUSSION THREAD

**Discussion by alissonrodrigo** [viniciussanchez/dataset-serialize#91] [Q&A] [answered]
Bom dia, antes de mais nada gostaria de agradeçer pela biblioteca e muito legal e uma mão na roda....

Tem como eu popular tables filhas vinda de um json?

[
  {
    "idtemplate": "1",
    "descricao": "Teste Dev",
    "idempresa": "3",
    "idcidade_origem": "8",
    "motoristas": [
      {
        "idmotorista": "8",
        "Motorista": "Alisson"
      },
      {
        "idmotorista": "9",
        "Motorista": "Rodrigo"
      }
    ],
    "cidades": [
      {
        "idcidade": "1",
        "cidade": "Capelinha"
      }
    ]
  }
]

Nesse exemplo ai queria popular a as querys Templares e suas filhas Motoristas e Cidades.
Grato

**Comment by viniciussanchez** [comment, marked as answer]
Sim, tem como.
Primeiramente, você terá uma query principal, e duas filhas, para fazer o carregamento dos motoristas e das cidades. Para isso você precisa criar um mestre detalhe entre os seus DataSet. Na pasta samples tem exemplos parecidos.
